compiler/rustc_resolve/src/macros.rs

```diff
@@ -788,6 +802,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
             self.report_out_of_scope_macro_calls(
                 ast_path,
                 parent_scope,
+                used_import,
```

Comment:
This is unnecessary, `invoc_in_mod_inert_attr.is_some()` means `used_import.is_none()`, you can add an assert.

Reply:
I will remove `used_import` in `report_out_of_scope_macro_calls`.